Packs/MicrosoftGraphMail/ReleaseNotes/1_5_18.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Microsoft Graph Mail Single User
+
+- Added `msgraph-mail-list-child-folders` command
```

Comment:
```suggestion
- Added the new `msgraph-mail-list-child-folders` command.
```
For more info see here
https://xsoar.pan.dev/docs/documentation/release-notes#general